services/repository-managers/src/main/java/org/sagebionetworks/repo/manager/table/TableQueryManagerImpl.java

```diff
@@ -520,42 +521,47 @@ public static void setDefaultValues(DownloadFromTableRequest request) {
 	 */
 	@Override
 	public DownloadFromTableResult runQueryDownloadAsStream(ProgressCallback progressCallback, UserInfo user,

```

Comment:
I would make this csv name specific and move the logic from the pfb worker here in a dedicated method